src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,22 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, fieldConfigs) => (values) => {
```

Comment:
you should not change the name of this, the entities was fine, because in most cases it's used with for the entities